modules/core/test/test_intrin_utils.hpp

```diff
@@ -554,6 +561,26 @@ template<typename R> struct TheTest
         return *this;
     }
 
+    // Handle accuracy for fp16
+    TheTest & test_div_fp16()
+    {
+#if CV_SIMD_FP16
```

Comment:
I propose to add skip message to else branch to make it obvious from test. FP16 support may be disabled.

Reply:
Using plain printf() or any fancy macros for this purpose?

Reply:
I do not have strong opinion, just to see that tests are skipped. It's very easy to loose fp16 support and have false impression that it's used and works well.

Reply:
Okay, added to everywhere needed.